vale/dictionaries/en_US-grafana.dic

```diff
@@ -1,13 +1,16 @@
-38
+42
 API/S po:noun
 Datadog/ po:noun
 ESLint/ po:noun
 Goldmark/ po:noun
 Grafana/M po:noun
+Jaeger/ po:noun
```

Comment:
FYI - Jaeger can also be an adjective (Jaeger Operator, Jaeger client, Jaeger agent)

Reply:
Great point! I think I have to add them as separate entries so will do that.

Reply:
I think there is a few other places in the word list where this needs to be added also so I've opened: https://github.com/grafana/writers-toolkit/pull/369